hazelcast.yaml

```diff
@@ -1,7 +1,361 @@
-apiVersion: hazelcast.com/v1alpha1
```

Comment:
What do you think about changing this file into hazelcast-full.yaml and keep existing yaml for easy to start. You can update readme to refer hazelcast-full.yaml if users want to customize their deployment.

Reply:
Good idea!